wolox-books/src/app/screens/auth/home/screens/book-list/components/home/home.component.ts

```diff
@@ -9,19 +10,28 @@ import { Book } from 'src/app/models/book';
 })
 
 export class HomeComponent implements OnInit {
-  constructor(private booksService: BooksService) { }
+  constructor(private booksService: BooksService, private cartCountService: CartCountService) { }
 
   searchBarPlaceHolderTxt: string = 'Search books by title...';
   filterName: string = 'filterBooks';
   filterBooks: any;
   currentPageContent: Book[];
+  cartCount: number;
```

Comment:
im not sure about having the cart on the home component. what about the other screens that show the cart?